00_Base/src/index.ts

```diff
@@ -77,7 +78,9 @@ export {
 } from './model/ClientInformation';
 export { ClientCredentialsRole } from './model/ClientCredentialsRole';
 export { fromCredentialsRoleDTO } from './model/ClientCredentialsRole';
-export { OcpiServerConfig } from './config/ocpi.server.config';
+export { ServerConfig } from './config/ServerConfig';
+export * from './config/sub';
```

Comment:
Why export everything from sub? Maybe we should have single setup for ENUMS, I think that's what is actually needed on the outside.

Reply:
The intent is to be able to import the sub classes as parts of the config into other classes where those config values are needed. You are correct in that this can be removed without any impact